src/code/views/share-dialog-view.tsx

```diff
@@ -12,6 +12,8 @@ const CFM_PRODUCTION_URL = "https://cloud-file-manager.concord.org"
 const CFM_PRODUCTION_AUTOLAUNCH_URL = `${CFM_PRODUCTION_URL}/autolaunch/autolaunch.html`
 const CODAP_PRODUCTION_URL = "https://codap.concord.org/releases/latest/"
 
+const FULLSCREEN_INTERACTIVE_URL = "https://models-resources.concord.org/question-interactives/version/v1.6.0/full-screen/?wrappedInteractive="
```

Comment:
Not for this PR, but it seems like there must be a better way to handle this sort of versioning problem. For instance, if this constant were exported from a `question-interactives` npm module then it could be managed the way other dependency versioning is handled.

Reply:
I ended up adding a story to do production builds of question-interactives at the root, it is covered in this story:

https://www.pivotaltracker.com/story/show/181270128